pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -181,6 +181,11 @@ type ClusterSpec struct {
 	// +nullable
 	Dashboard DashboardSpec `json:"dashboard,omitempty"`
 
+	// Prometheus settings
+	// +optional
+	// +nullable
+	Prometheus PrometheusSpec `json:"prometheus,omitempty"`
```

Comment:
How about moving this inside the MonitoringSpec? It already has a setting for enabled/disabled, and is purely about prometheus monitoring.